index.js

```diff
@@ -1445,6 +1445,9 @@ Buffer.prototype.writeDoubleBE = function writeDoubleBE (value, offset, noAssert
 
 // copy(targetBuffer, targetStart=0, sourceStart=0, sourceEnd=buffer.length)
 Buffer.prototype.copy = function copy (target, targetStart, start, end) {
+  if (target === undefined) {
```

Comment:
Better to check if the argument is not actually a Buffer to catch all possible bad values instead of just `undefined`